app/assets/javascripts/components/story/StoryNotes.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Note from 'components/notes/Note';
+import NoteForm from 'components/notes/NoteForm'
+
+class StoryNotes extends React.Component {
+  constructor() {
+    super();
+
+    this.renderNotes = this.renderNotes.bind(this);
+    this.renderStoryNotes = this.renderStoryNotes.bind(this);
+  }
+
+  render() {
```

Comment:
If you follow my other comment (that has a link to this line) you can rename `renderStoryNotes` to just `render` and remove this method.